src/dotnet/scripts/dotnet-helpers.sh

```diff
@@ -117,3 +115,22 @@ install_runtime() {
         --install-dir "$DOTNET_INSTALL_DIR" \
         --no-path
 }
+
+# Installs a .NET workload
+# Usage: install_workload <workload_id>
+# Reference: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-install
+install_workload() {
+    local workload_id="$1"
+
+    # Prevent "Welcome to .NET" message from dotnet
+    local DOTNET_NOLOGO=true
+
+    # Prevent generating a development certificate
+    local DOTNET_GENERATE_ASPNET_CERTIFICATE=false
+
+    echo "Executing dotnet workload install $workload_id --temp-dir /tmp/dotnet-workload-temp-dir"
+    dotnet workload install "$workload_id" --temp-dir /tmp/devcontainer-nuget-cache
```

Comment:
Should this be `/tmp/dotnet-workload-temp-dir` to fix the tests?

Reply:
![JimcareyDuhGIF](https://github.com/devcontainers/features/assets/1583241/83413e79-beb1-4914-ac4f-af065a70439f)

Reply:
Fixed it, thanks for the hint